distros/debian/buster/Dockerfile.base

```diff
@@ -5,5 +5,6 @@ RUN apt-get -qq update && \
     apt-get install -y -qq curl ca-certificates build-essential \
                            cmake make bash sudo wget unzip dh-make \
                            libsystemd-dev zlib1g-dev flex bison \
-                           libssl1.1 libssl-dev libpq-dev postgresql-server-dev-all && \
+                           libssl1.1 libssl-dev libpq-dev postgresql-server-dev-all \
```

Comment:
Not strictly required. I'm only using it on X86, but I figured that it would be desirable to have feature parity across all architectures. Without openssl specifically, some valid TLS certs fail to be parsed, which causes issues. If the idea is that ARM builds should be more lightweight, maybe have different versions of the ARM builds available?

Reply:
Oops - forgot to add for the Debian and Raspbian builds - added